packages/ember-htmlbars/tests/helpers/if_unless_test.js

```diff
@@ -51,248 +47,6 @@ QUnit.module('ember-htmlbars: {{#if}} and {{#unless}} helpers', {
-  view = EmberView.create({
-    array: ArrayProxy.create({ content: emberA() }),
-
-    template: compile('{{#unless view.array}}Yep{{/unless}}')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'Yep');
-
-  run(function() {
-    view.get('array').pushObject(1);
-  });
-
-  equal(view.$().text(), '');
-
-  run(function() {
-    view.get('array').removeObject(1);
-  });
-
-  equal(view.$().text(), 'Yep');
-});
-
-QUnit.test('The `if` helper updates when the value changes', function() {
-  view = EmberView.create({
-    conditional: true,
-    template: compile('{{#if view.conditional}}Yep{{/if}}')
-  });
-  runAppend(view);
-  equal(view.$().text(), 'Yep');
-  run(function() {
-    view.set('conditional', false);
-  });
-  equal(view.$().text(), '');
-});
-
-QUnit.test('The `if (unbound` helper does not update when the value changes', function() {
-  view = EmberView.create({
-    conditional: true,
-    template: compile('{{#if (unbound view.conditional)}}Yep{{/if}}')
-  });
-  runAppend(view);
-  equal(view.$().text(), 'Yep');
-  run(function() {
-    view.set('conditional', false);
-  });
-  equal(view.$().text(), 'Yep');
-});
-
-QUnit.test('The `unless` helper updates when the value changes', function() {
-  view = EmberView.create({
-    conditional: false,
-    template: compile('{{#unless view.conditional}}Nope{{/unless}}')
-  });
-  runAppend(view);
-  equal(view.$().text(), 'Nope');
-  run(function() {
-    view.set('conditional', true);
-  });
-  equal(view.$().text(), '');
-});
-
-QUnit.test('The `if (unbound` helper does not update when the value changes', function() {
-  view = EmberView.create({
-    conditional: false,
-    template: compile('{{#unless (unbound view.conditional)}}Nope{{/unless}}')
-  });
-  runAppend(view);
-  equal(view.$().text(), 'Nope');
-  run(function() {
-    view.set('conditional', true);
-  });
-  equal(view.$().text(), 'Nope');
-});
-
-QUnit.test('The `if (unbound` helper should work when its inverse is not present', function() {
-  view = EmberView.create({
-    conditional: false,
-    template: compile('{{#if (unbound view.conditional)}}Yep{{/if}}')
-  });
-  runAppend(view);
-  equal(view.$().text(), '');
-});
-
-QUnit.test('should not rerender if truthiness does not change', function() {
-  view = EmberView.create({
-    template: compile('<h1 id="first">{{#if view.shouldDisplay}}{{view view.InnerViewClass}}{{/if}}</h1>'),
-
-    shouldDisplay: true,
-
-    InnerViewClass: EmberView.extend({
-      template: compile('bam')
-    })
-  });
-
-  runAppend(view);
-
-  equal(view.$('#first').text(), 'bam', 'renders block when condition is true');
-  equal(view.$('#first div').text(), 'bam', 'inserts a div into the DOM');
-
-  run(function() {
-    set(view, 'shouldDisplay', 1);
-  });
-
-  equal(view.$('#first').text(), 'bam', 'renders block when condition is true');
-});
-
-QUnit.test('should update the block when object passed to #unless helper changes', function() {
```

Comment:
Removed without replacement (?). Seems like this is about the content of the bock get updated? I can add it back if it is important.